File: Archiver/frmLocalParameters.vb
Public Class frmLocalParameters

    Sub New()
        ' This call is required by the Windows Form Designer.
        InitializeComponent()

    End Sub

    Sub LoadParms()

    End Sub

    Function UpdateParm() As Boolean
        Dim B As Boolean = True
        Try

        Catch ex As Exception
            B = False
        End Try
        Return B
    End Function

    Function DeleteParm() As Boolean
        Dim B As Boolean = True
        Try

        Catch ex As Exception
            B = False
        End Try
        Return B
    End Function

    Function AddParm() As Boolean
        Dim B As Boolean = True
        Try

        Catch ex As Exception
            B = False
        End Try
        Return B
    End Function

    Private Sub DataGridView1_CellContentClick(sender As Object, e As DataGridViewCellEventArgs) Handles dgParms.CellContentClick

        Try

        Catch ex As Exception
            Console.WriteLine(ex.Message)
        End Try

    End Sub
End Class

File: Archiver/frmLocalParameters.cs
using System;
using System.Windows.Forms;
using MODI;

namespace EcmArchiver
{
    public partial class frmLocalParameters
    {
        public frmLocalParameters()
        {
            // This call is required by the Windows Form Designer.
            InitializeComponent();
            _dgParms.Name = "dgParms";
        }

        public void LoadParms()
        {
        }

        public bool UpdateParm()
        {
            bool B = true;
            try
            {
            }
            catch (Exception ex)
            {
                B = false;
            }

            return B;
        }

        public bool DeleteParm()
        {
            bool B = true;
            try
            {
            }
            catch (Exception ex)
            {
                B = false;
            }

            return B;
        }

        public bool AddParm()
        {
            bool B = true;
            try
            {
            }
            catch (Exception ex)
            {
                B = false;
            }

            return B;
        }

        private void DataGridView1_CellContentClick(object sender, DataGridViewCellEventArgs e)
        {
            try
            {
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
            }
        }
    }
}